extensions/PhabBugz/lib/Util.pm

```diff
@@ -64,7 +65,13 @@ sub set_attachment_approval_flags {
     'needs-revision'  => '-',
   };
 
-  my $status = $revision_status_flag_map->{$revision->status};
+  # Find the current review status of the revision changer
+  my $status = undef;
+  foreach my $reviewer (@{$revision->reviews}) {
+    if ($reviewer->id == $phab_user->id) {
+      $status = $reviewer->status;
```

Comment:
Question: looking at `extensions/PhabBugz/lib/Revision.pm`, it seems we have a custom `accepted-prior` status that we use when the revision has been accepted on a diff that is not the latest. Do you think we should add `accepted-prior` to the `$revision_status_flag_map`, with the same `+` BMO flag as `accepted`?

I would like to avoid having the flag be set to `-`, or unset, when a release manager has approved the revision already.